src/NodeVariableService.cpp

```diff
@@ -70,10 +70,12 @@ void NodeVariableService::handleReadNV(const VlcbMessage *msg, unsigned int nn)
   if (nvindex == 0)
   {
     controller->sendMessageWithNN(OPC_NVANS, nvindex, module_config->EE_NUM_NVS);
-
-    for (int i = 1; i <= module_config->EE_NUM_NVS; ++i)
+    if (!controller->getModuleConfig()->fcuCompatible)
```

Comment:
You can replace "controller->getModuleConfig()" with "module_config".

Reply:
Of course! I'm not sure why I did that. I'll change it